src/open-data/Services.csv

```diff
@@ -1,314 +1,314 @@
+Microsoft.DesktopVirtualization;microsoft.desktopvirtualization/hostpools;Azure Virtual Desktop;Compute;Microsoft;Cloud Provider;Cloud;IaaS
+microsoft.devcenter;microsoft.devcenter/projects;Microsoft Dev Box;Developer Tools;Microsoft;Cloud Provider;Cloud;PaaS
+microsoft.devices;microsoft.devices/iothubs;Azure IoT Hub;Internet of Things;Microsoft;Cloud Provider;Cloud;PaaS
+Microsoft.EventGrid;microsoft.devices/iothubs;Azure IoT Hub;Internet of Things;Microsoft;Cloud Provider;Cloud;PaaS
+MICROSOFT.DEVICES;microsoft.devices/provisioningservices;Azure IoT Hub;Internet of Things;Microsoft;Cloud Provider;Cloud;PaaS
+Microsoft.DeviceUpdate;microsoft.deviceupdate/accounts;Azure IoT Hub;Internet of Things;Microsoft;Cloud Provider;Cloud;PaaS
+Microsoft.DigitalTwins;microsoft.digitaltwins/digitaltwinsinstances;Azure Digital Twins;Internet of Things;Microsoft;Cloud Provider;Cloud;PaaS
+Microsoft.DocumentDB;microsoft.documentdb/cassandraclusters;Cosmos DB;Databases;Microsoft;Cloud Provider;Cloud;PaaS
+microsoft.documentdb;microsoft.documentdb/databaseaccounts;Cosmos DB;Databases;Microsoft;Cloud Provider;Cloud;PaaS
+Microsoft.DocumentDB;microsoft.documentdb/mongocluster;Cosmos DB;Databases;Microsoft;Cloud Provider;Cloud;PaaS
+Microsoft.DocumentDB;microsoft.documentdb/mongoclusters;Cosmos DB;Databases;Microsoft;Cloud Provider;Cloud;PaaS
+microsoft.domainregistration;microsoft.domainregistration/domains;App Service;Web;Microsoft;Cloud Provider;Cloud;PaaS
+Microsoft.Easm;microsoft.easm/workspaces;Microsoft Defender for Cloud;Security;Microsoft;Cloud Provider;Cloud;SaaS
+Microsoft.EdgeOrder;microsoft.edgeorder/orderitems;Azure Stack Edge;Multicloud;Microsoft;Cloud Provider;Cloud;IaaS
+Microsoft.ElasticSan;microsoft.elasticsan/elasticsans;Azure Elastic SAN;Storage;Microsoft;Cloud Provider;Cloud;SaaS
+Microsoft.EventGrid;microsoft.eventgrid/domains;Event Grid;Integration;Microsoft;Cloud Provider;Cloud;PaaS
+Microsoft.EventGrid;microsoft.eventgrid/extensiontopics;Event Grid;Integration;Microsoft;Cloud Provider;Cloud;PaaS
+Microsoft.EventGrid;microsoft.eventgrid/namespaces;Event Grid;Integration;Microsoft;Cloud Provider;Cloud;PaaS
+MICROSOFT.EVENTGRID;microsoft.eventgrid/partnertopics;Event Grid;Integration;Microsoft;Cloud Provider;Cloud;PaaS
+MICROSOFT.EVENTGRID;microsoft.eventgrid/systemtopics;Event Grid;Integration;Microsoft;Cloud Provider;Cloud;PaaS
+Microsoft.EventGrid;microsoft.eventgrid/topics;Event Grid;Integration;Microsoft;Cloud Provider;Cloud;PaaS
+Microsoft.EventHub;microsoft.eventhub/clusters;Event Hubs;Integration;Microsoft;Cloud Provider;Cloud;PaaS
+Microsoft.EventGrid;microsoft.eventhub/namespaces;Event Hubs;Integration;Microsoft;Cloud Provider;Cloud;PaaS
+Microsoft.EventHub;microsoft.eventhub/namespaces;Event Hubs;Integration;Microsoft;Cloud Provider;Cloud;PaaS
+Microsoft.Fabric;microsoft.fabric/capacities;Microsoft.Fabric;Analytics;Microsoft;Cloud Provider;Cloud;SaaS

```

Comment:
This is a tricky one... Power BI is SaaS, but Fabric is a data platform and all the tools in Fabric are PaaS. Does merging them together make it SaaS? Or is it still PaaS as a data engineering solution? I'm fine with SaaS. Just thinking out loud in case anyone has a more firm opinion.